package.json

```diff
@@ -25,13 +25,14 @@
     "build:all": "npm run clean && npm run build && npm run test:noschema && npm run build:images",
     "build:images": "npm run data && scripts/generate-images.sh",
     "build:toc": "bundle exec jekyll build --incremental -q && scripts/generate-toc",
+    "build:versions": "scripts/update-version.sh",
     "build:test-gallery": "browserify test-gallery/main.ts -p [tsify -p test-gallery] -d > test-gallery/main.js",
     "checkschema": "./scripts/check-schema.sh",
     "clean": "rm -f vega-lite.* & find -E src test site -regex '.*\\.(js|js.map|d.ts)' -delete & rm -rf examples/_diff examples/_original examples/_output examples/images && rm -rf data",
     "data": "rsync -r node_modules/vega-datasets/data/* data",
     "deploy": "scripts/deploy.sh",
     "deploy:gh": "scripts/deploy-gh.sh",
-    "deploy:schema": "scripts/deploy-schema.sh",
+    "deploy:schema": "./scripts/deploy-schema.sh",
```

Comment:
No problem. I realize I reviewed code before you said you were done but I thought it would be useful for you to get early feedback.